codecs/text/src/test/java/com/datastax/oss/dsbulk/codecs/text/json/JsonNodeToVectorCodecTest.java

```diff
@@ -104,4 +104,16 @@ void should_encode_too_many_but_not_too_few() {
     assertThatThrownBy(() -> dsbulkCodec.encode(tooFewNode, ProtocolVersion.DEFAULT))
         .isInstanceOf(IllegalArgumentException.class);
   }
+
+  /* Issue 484: now that we're using the dsbulk string-to-subtype converters we should get
+   * enforcement of existing dsbulk policies.  For our purposes that means the failure on
+   * arithmetic overflow */
+  @Test
+  void should_not_convert_too_much_precision() {
+    ArrayNode tooPreciseNode = JSON_NODE_FACTORY.arrayNode();
+    tooPreciseNode.add(JSON_NODE_FACTORY.numberNode(6.646329843));
+    assertThat(dsbulkCodec).cannotConvertFromInternal(tooPreciseNode);
```

Comment:
Hmm I don't get why you are trying to use `dsbulkCodec` to convert from an _internal_ type that would be... `JsonNode`? 

The internal type of this codec is `CqlVector`, so calling `cannotConvertFromInternal` would make sense only
if you had some instance of `CqlVector` that is somehow "invalid" – maybe a `CqlVector` with the wrong number of dimensions, or something like that.

But calling `cannotConvertFromInternal(tooPreciseNode)` does not make sense to me. It's only possible, btw, because `dsbulkCodec` is of the raw type `JsonNodeToVectorCodec`. It should be `JsonNodeToVectorCodec<Float>` – but in that case, I bet this statement wouldn't compile anymore.

If you are trying to check whether the _external_ type `tooPreciseNode` causes a runtime error, then call `cannotConvertFromExternal(tooPreciseNode)` – I fixed something similar for `StringToVectorCodecTest`, see test `should_not_convert_from_invalid_external`.